sway-lib-std/src/address.sw

```diff
@@ -102,3 +103,9 @@ impl Address {
         self.transfer(amount, contract_id());
     }
 }
+
+impl Hash for Address {
+    fn hash(self, ref mut state: Hasher) {
+        self.value.hash(state);
```

Comment:
It would be safer always to deconstruct the struct, right? And let the compiler complains if the implementation change.

```rust
fn hash(self, ref mut state: Hasher) {
    let Address { value } = self;
    value.hash(state);
}
```

Reply:
Fixed, thanks.